src/messages.js

```diff
@@ -216,7 +216,7 @@ var warnings = {
   W134: "The '{a}' option is only available when linting ECMAScript {b} code.",
   W135: "{a} may not be supported by non-browser environments.",
   W136: "'{a}' must be in function scope.",
-  W137: "Empty destructuring."
+  W137: "Empty {a}."
```

Comment:
On another PR it was pointed out one of my messages was too simplistic and we could be more helpful. I am wondering if thats the case here, but I can't think of a more useful message - it seems pretty straightforward?

Empty destructuring, this is un-necessary and can be removed? I'm not sure, so only make that change if you agree.